app_packs/alerts/README.md

```diff
@@ -0,0 +1,95 @@
+# Automatically Create Alert App Packs Via Script
+
+## Summary:
+
+#### The essentials
+- Run this API script to create Alert App Pack rules for projects within an org.
+- :warning: By default, it will create alerts for every project in an org.
+- You can also pass a list of projects to create alerts for (see Run Script section).
+  
+#### Other things to know
+- The alerts are created without being assigned any teams. SEs or customers will need to assign teams.
+- The alerts are created with zero actions. SEs will need to encourage customers to add actions (i.e. set up an email/slack notification)
```

Comment:
- The alerts are created with zero actions. SEs will need to encourage customers to add actions (i.e. set up an email/slack notification)





[3:41](https://sentry.slack.com/archives/D02AMS00GH0/p1667428904696209)
I would change this to add actions and further personalize, SEs should use this as a point of engagement with customers
[3:41](https://sentry.slack.com/archives/D02AMS00GH0/p1667428907322439)
something like that
[3:41](https://sentry.slack.com/archives/D02AMS00GH0/p1667428917944749)
basically rehashing the Notion stuff you added but just a bit